### PR TITLE
Support command line options

### PR DESCRIPTION
**Pull request by DavidPoliakoff**
So, Kokkos has added the ability to pass command-line arguments to tools. This just adds support for that functionality, taking a single argument and converting it into an argument to be passed to a ConfigManager

### CODE REVIEW

File: src/services/kokkos/KokkosProfilingSymbols.cpp [hunk 1]
```diff
@@ -1,68 +1,100 @@
...
 #include "caliper/Annotation.h"
+#include "caliper/ConfigManager.h"
 #include "caliper/cali.h"
-
+#include "types.hpp"
+#include <cstdint>
 cali::kokkos::callbacks kokkosp_callbacks;
 
 using cali::kokkos::SpaceHandle;
+cali::ConfigManager mgr;
```
Comment (daboehme) [inline src/services/kokkos/KokkosProfilingSymbols.cpp:9]: The `mgr` object should be in the `cali::kokkos` namespace.

**Review by daboehme — CHANGES_REQUESTED** [review]
Let's put the `mgr` object into a proper namespace, otherwise looks good!

**Comment by jrmadsen** [comment]
Might want to also add the print help function to at least say "hey, you can pass in config manager params here"

**Comment by DavidPoliakoff** [comment]
@daboehme , is there a "get me a list of config manager configurations" guy I can use for that?

**Comment by daboehme** [comment]
@DavidPoliakoff there is `ConfigManager::available_config_specs()` and `ConfigManager::get_documentation_for_spec(const char*)`. See https://software.llnl.gov/Caliper/ConfigManagerAPI.html#_CPPv4NK4cali13ConfigManager22available_config_specsEv

**Comment by DavidPoliakoff** [comment]
@daboehme : sorry for the slow turnaround on that

**Review by daboehme — APPROVED** [review]

**Pull request merged**